sanic/pages/error.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Mapping
+
+import tracerite.html  # type: ignore
+
+from html5tagger import E
+from tracerite import html_traceback, inspector  # type: ignore
+
+from sanic.request import Request
+
+from .base import BasePage
+
+
+# Avoid showing the request in the traceback variable inspectors
+inspector.blacklist_types += (Request,)
+
+ENDUSER_TEXT = """We're sorry, but it looks like something went wrong. Please try refreshing the page or navigating back to the homepage. If the issue persists, our technical team is working to resolve it as soon as possible. We apologize for the inconvenience and appreciate your patience."""  # noqa: E501
```

Comment:
One super long line with a `noqa` makes this difficult to maintain. We can break this up and make it easier to read using `dedent` if needed, etc.